### DISCUSSION TITLE
Show all pictures from folder or list

### DISCUSSION THREAD

**Discussion by JacekKolcon** [fengyuanchen/viewerjs#542] [Ideas]
Currently, a slide show is only possible from the list of loaded images. I suggest displaying a slide show from a given list and / or folder.

For example, you can add a function that, after object initialization, will allow the list of images to be fed into another url.

vieverObj.AddUrl ('/ pic / pic001.jpg', ...);
viewerObj.UseFolder ('/ pic');

Congratulations, your code is great.

**Comment by fengyuanchen** [comment]
Currently, you can destroy the existing viewer and initialize a new one in another list and/or folder.

**Reply by JacekKolcon** [reply]
Thank you for your answer.
I still don't know how to use a folder or a gallery list without uploading pictures to the website. Is it possible?

**Reply by fengyuanchen** [reply]
I'm not sure about that.
